java/impl/src/main/java/com/example/saas/step6usagereporting/Report.java

```diff
@@ -47,15 +50,11 @@ public static void main(String[] args) throws Exception {
 
     String serviceName = args[0];
 
-    ServiceControl serviceControl =
-        new ServiceControl.Builder(
-                GoogleNetHttpTransport.newTrustedTransport(),
-                new JacksonFactory(),
-                new HttpCredentialsAdapter(
-                    GoogleCredentials.getApplicationDefault().createScoped(
-                        Collections.singletonList(CLOUD_SCOPE))))
-            .setApplicationName(ServiceOptions.getDefaultProjectId())
-            .build();
+    ServiceControl serviceControl = new ServiceControl.Builder(GoogleNetHttpTransport.newTrustedTransport(),

```

Comment:
Don't see any difference. Is it just the format?

Reply:
Yup